snapcraft/_store.py

```diff
@@ -32,7 +32,7 @@
 logger = logging.getLogger(__name__)
 
 
-def _get_name_from_snap_file(snap_path):
+def _get_data_from_snap_file(snap_path):
```

Comment:
This change affects push & sign-build, so it's covered indirectly, however I think it deserves a unittest (and dropping the undescore)